### PR TITLE
Merge branch `beta` into `main`

### PR DESCRIPTION
**Pull request by kdev**

### CODE REVIEW

File: .github/workflows/development.yml [hunk 1]
```diff
@@ -57,12 +57,12 @@ jobs:
...
             - name: '☁️ checkout repository'
-              uses: actions/checkout@v3
+              uses: actions/checkout@v4
 
             - name: '🔧 Setup Node.js'
-              uses: actions/setup-node@v3
+              uses: actions/setup-node@v4
               with:
-                  node-version: 'lts/*'
+                  node-version: '18'
```
Comment (jxn-30) [inline .github/workflows/development.yml:65]: is there a reason for using 18 instead of 20?

Comment (kdev) [reply .github/workflows/development.yml:65]: Erm no not really, gonna revert that to "lts/*"

**Comment by kdev** [comment]
Ah I forgot: we cannot use node 20 since libcurl has no prebuilt binaries so we need to stay with node 18 for now.
I am going to revert to node 18

**Comment by jxn-30** [comment]
> Ah I forgot: we cannot use node 20 since libcurl has no prebuilt binaries so we need to stay with node 18 for now. I am going to revert to node 18

Ah right, having the same issue on another repo. FTR tracking issue of `node-libcurl`: https://github.com/JCMais/node-libcurl/issues/399

**Comment by kdev** [comment]
> [JCMais/node-libcurl#399](https://github.com/JCMais/node-libcurl/issues/399)

I've subscribed to that issue. I will update to node 20 when the issue is resolved by the owner. Until then we need to stick with node 18.

**Comment by kdev** [comment]
@jxn-30 I think we are ready for review

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**